From 8e1c4b354cdc01f32d7903fbd7564c2ea2a5a7a1 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 9 Nov 2015 15:33:27 +0100
Subject: [PATCH] dracut-logger: prefix stderr output with "dracut: "

dnf updates of the kernel execute dracut in %post.
For the user it is not possible to distinguish the dracut output from
other error messages.
---
 dracut-logger.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-logger.sh b/dracut-logger.sh
index e5c9358..d1cee98 100755
--- a/dracut-logger.sh
+++ b/dracut-logger.sh
@@ -324,7 +324,7 @@ _do_dlog() {
     local msg="$*"
     local lmsg="$lvlc: $*"
 
-    (( $lvl <= $stdloglvl )) && echo "$msg" >&2
+    (( $lvl <= $stdloglvl )) && printf -- 'dracut: %s\n' "$msg" >&2
 
     if (( $lvl <= $sysloglvl )); then
         if [[ "$_dlogfd" ]]; then
